src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -79,15 +79,22 @@ class SQLTranslatorTest : public BaseTest {
     int_int_int_c = stored_table_node_int_int_int->get_column("c");
   }
 
-  std::shared_ptr<opossum::AbstractLQPNode> compile_query(const std::string& query,
-                                                          const UseMvcc use_mvcc = UseMvcc::No) {
+  std::shared_ptr<opossum::AbstractLQPNode> compile_query(
+      const std::string& query, const UseMvcc use_mvcc = UseMvcc::No,
+      const std::shared_ptr<SQLTranslationInfo>& translation_info = nullptr) {
     hsql::SQLParserResult parser_result;
     hsql::SQLParser::parseSQLString(query, &parser_result);
     Assert(parser_result.isValid(), create_sql_parser_error_message(query, parser_result));
 
     const auto translation_result = SQLTranslator{use_mvcc}.translate_parser_result(parser_result);
     const auto lqps = translation_result.lqp_nodes;
 
+    if (translation_info) {
+      translation_info->cacheable = translation_result.translation_info.cacheable;
+      translation_info->parameter_ids_of_value_placeholders =
+          translation_result.translation_info.parameter_ids_of_value_placeholders;
+    }
+
```

Comment:
Any reason you can't return a pair instead of having an output parameter?

Reply:
Nope. Just that we have to change all tests then.

Reply:
Looks like 90% can be covered with a single Regexp